### ISSUE TITLE
Support ubuntu 20.04 in vagrant setup script

### ISSUE THREAD

**Issue by Phantomape** [spdk/spdk#1680]
## Expected Behavior

Vagrant setup script should work under 20.04

## Current Behavior

For now, the vagrant setup script does not support ubuntu 20.04, only 16.04 and 18.04 are supported.

Should have had 20.04 supported

## Possible Solution

Support 20.04 in the near future

**Comment by wawryk** [comment]
@Phantomape 
Please try [this patch](https://review.spdk.io/gerrit/c/spdk/spdk/+/5109)

**Comment by yidong0635** [comment]
Could you please update the README.md? @wawryk

**Comment by Phantomape** [comment]
I'm able to start the vm on ubuntu 20.04. However, when I tried to run the hello-world example, it failed. I took a closer look into the script, it seems that the command ```lspci -mm -n -D | grep -i -- "-p02" | awk -v 'cc="0108"' -F ' ' '{if (cc ~ $2) print $1}'``` in function ```iter_all_pci_class_code```(common.sh) exit silently. The output of command ```lspci -mm -n -D``` is attached below
![image](https://user-images.githubusercontent.com/8809221/99360003-dcfda200-28ea-11eb-9a6e-2a08b7930c6e.png)
And I don't see any string starting with ```-p02``` in the output, which is why the command outputs nothing
